docs/wiki/User Guide/REST-APIs.md

```diff
@@ -603,4 +604,23 @@ To drop recently removed/demoted brokers, send POST request like:
 
 ### 2-step Verification
 
-2-step verification aims to help users verify the command they (or their peers) intend to run by letting them review requests explicitly to approve or discard them, and enable execution of only the approved requests. Read [2 step verification for POST requests](https://github.com/linkedin/cruise-control/wiki/2-step-verification-for-POST-requests) for more detail.
\ No newline at end of file
+2-step verification aims to help users verify the command they (or their peers) intend to run by letting them review requests explicitly to approve or discard them, and enable execution of only the approved requests. Read [2 step verification for POST requests](https://github.com/linkedin/cruise-control/wiki/2-step-verification-for-POST-requests) for more detail.
+
+### Rightsize the cluster with the Provisioner
+The following POST request can create a request to the provisioner to rightsize the broker or partition of a cluster.
+
+    POST /kafkacruisecontrol/rightsize
+
+Supported parameters are:
+
+| PARAMETER                           | TYPE      | DESCRIPTION                                                                       | DEFAULT   | OPTIONAL  |
+|-------------------------------------|-----------|-----------------------------------------------------------------------------------|-----------|-----------|
+| num_brokers_to_add                  | integer   | difference in broker count to rightsize towards                                   | -1        | yes       |
+| partition_count                     | integer   | target number of partitions to rightsize towards                                  | -1        | yes       |
+| topic                               | regex     | regular expression to specify subject topics                                      | null      | yes       | 
+| allow_capacity_estimation           | boolean   | whether to allow broker capacity to be estimated from other broker in the cluster | true      | yes       |
+| doAs                                | string    | propagated user by the trusted proxy service                                      | null      | yes       |
+| exclude_recently_demoted_brokers    | boolean   | whether to allow leader replicas to be moved to recently demoted brokers          | false     | yes       |
+| get_response_schema                 | boolean   | return JSON schema in response header or not                                      | false     | yes       |
+| json                                | boolean   | return in JSON format or not                                                      | false     | yes       |
+| verbose                             | boolean   | return detailed state information                                                 | false     | yes       |
```

Comment:
is `verbose` a parameter of this endpoint?

Reply:
Removed `verbose`.